src/hooks/useQuery/index.js

```diff
@@ -0,0 +1,5 @@
+import { useLocation } from 'react-router-dom';
+
+export const useQuery = () => {
```

Comment:
can be this hook named as useQueryParams to be more especific? :)

Reply:
Perfect!